ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -105,7 +103,9 @@
 
   private final AtomicBoolean started = new AtomicBoolean(false);
 
-  private final PeerDiscoveryAgent agent;
+  private final SECP256K1.KeyPair keypair;
+  final DiscoveryPeer self;
```

Comment:
Not a fan of `self` outside python (nor of `this` inside of python).  Could this become `discoveryPeer` or `localPeer` or something else?  

My concern is that it maps to the same concept and so I keep reading it as `this`.